chucks-pick3-web/angular-app/src/app/components/info-view.component/info-view.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="how-to-header" id="generated-picks">
   <h1><b>How to View Generated Picks:</b></h1>
 </div>
-  <div class="how-to-view">
-  <img src="/assets/images/Info page -14.png" alt="ViewGenerated" style="width:700px" align="center">
+<div class="how-to-view">
+  <img src="src/app/assets/images/Info page -14.png" alt="ViewGenerated" style="width:700px" align="center">
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted to assets/images